docs/getting-started/_common/os-support-info.rst

```diff
@@ -1,15 +1,15 @@
 You can `build ScyllaDB from source <https://github.com/scylladb/scylladb#build-prerequisites>`_ on other x86_64 or aarch64 platforms, without any guarantees.
 
-+----------------------------+-------------+---------------+---------+---------------+
-| Linux Distributions        |Ubuntu       | Debian        | CentOS /| Rocky /       |
-|                            |             |               | RHEL    | RHEL          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+
-| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   7     |   8   |   9   |
-+============================+======+======+=======+=======+=========+=======+=======+
-|   5.4                      | |v|  | |v|  | |v|   | |v|   | |x|     | |v|   | |v|   |
-+----------------------------+------+------+-------+-------+---------+-------+-------+
-|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+-------+-------+---------+-------+-------+
++----------------------------+-------------+---------------+---------------+
+| Linux Distributions        |Ubuntu       | Debian        | Rocky /       |
+|                            |             |               | RHEL          |
++----------------------------+------+------+-------+-------+-------+-------+
+| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   8   |   9   |
++============================+======+======+=======+=======+=======+=======+
+|   6.0                      | |v|  | |v|  | |v|   | |v|   | |v|   | |v|   |
```

Comment:
@mykaul Should we try to get `24.04` in for 6.0? or later releases only??

Reply:
We could - once we test it. I don't believe we have done it yet.